fedimint-bitcoind/src/bitcoincore_rpc.rs

```diff
@@ -75,6 +77,11 @@ pub fn make_electrum_rpc(url: &Url, _task_handle: TaskHandle) -> Result<DynBitco
     Ok(ElectrumClient::new(url)?.into())
 }
 
+// TODO: (@oleonardolima) Should we use a retry mechanism for esplora client ?
```

Comment:
IIRC `ElectrumClient` already had one, so I skipped it. Check if `EsploraClient` has one too. 

We should probably unify the behavior there and just keep trying forever for all clients.

Reply:
I don't think we need to re-try client creation, I didn't see any possibly transient error cases. After it's created we can just wrap it in a retry client.

Reply:
> I don't think we need to re-try client creation, I didn't see any possible transient error cases. After it's created we can just wrap it in a retry client.

Agreed, as dpc mentioned it probably should unify and wrap it in a retry client. After checking it on esplora crate I'll just wrap it in a retry client.